src/detect-tls-random.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright (C) 2022 Open Information Security Foundation
+ *
+ * You can copy, redistribute or modify this Program under the terms of
+ * the GNU General Public License version 2 as published by the Free
+ * Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * version 2 along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+ * 02110-1301, USA.
+ */
+
+#ifndef __DETECT_TLS_RANDOM_H__
+#define __DETECT_TLS_RANDOM_H__
+
+#include "app-layer-ssl.h"
+
+void DetectTlsRandomRegister(void);
+uint32_t DetectTlsRandomInspectMpm(DetectEngineThreadCtx *det_ctx, SSLState *ssl_state);
```

Comment:
unused?

Reply:
Indeed. A leftover. Shall remove it in rev 2 after Juliana's review on docs.